Makefile

```diff
@@ -55,7 +55,16 @@ dev-fix: dev-setup
 	$(VENV)/flynt -tc -v .
 
 doc-gen:
-	$(VENV)/python tools/doc/plugin_doc_gen.py
+	python3 -m tools.doc.plugin_doc_gen
```

Comment:
can we keep using `$(VENV)/python`?

Reply:
Yes. I made this change only because CI environment does not set up venv and check-doc-gen needs doc-gen.
Maybe we can keep using `$(VENV)/python` in doc-gen and use `python3 -m tools.doc.plugin_doc_gen` instead of `make doc-gen` in check-doc-gen.
What do you think?

Reply:
@JarvisG495 let's add a goal in Makefile

```makefile
$(VENV):
	python3 -m venv $(VENV_DIR)
```

And add dependent `$(VENV)` to all other goals that use `$(VENV)`, so that it `venv` can  be initialized automatically